protobufs/livekit_ingress.proto

```diff
@@ -113,6 +113,9 @@ message IngressVideoEncodingOptions {
   repeated VideoLayer layers = 3;
 }
 
+message IngressVideoEncodingPassthrough {
```

Comment:
what's this for?

Reply:
Leftover from a previous version. I removed it. Thanks.